database/migrations/2023_07_25_062017_remove_name_add_first_name_last_name_to_users_table.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class RemoveNameAddFirstNameLastNameToUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->dropColumn('name');
```

Comment:
```suggestion
            $table->string('first_name')->nullable();
            $table->string('last_name')->nullable();
            $table->dropColumn('name');
```
Viết chung được vào 1 function mà e, k cần tách ra